types/mongodb/index.d.ts

```diff
@@ -1371,7 +1371,7 @@ export interface Collection<TSchema extends { [key: string]: any } = DefaultSche
     ): Promise<TSchema | null>;
     findOne<T = TSchema>(
         filter: FilterQuery<TSchema>,
-        options: FindOneOptions<T extends TSchema ? TSchema : T>,
+        options?: FindOneOptions<T extends TSchema ? TSchema : T>,
```

Comment:
You should add a test calling this function without the `options` parameter to prevent future regression

Reply:
Right, that's why I made this small edit to the existing test [here](https://github.com/DefinitelyTyped/DefinitelyTyped/pull/51073/files#diff-6b5da2618493adf61d45da47ff1a6f6721cea888c0d7e94a5e8754988ad6d342R84)

There is then coverage for:
- `findOne` with no generic and no options
- `findOne` with a generic and no options
- `findOne` with a generic and options

What did I miss?

Reply:
My bad

Reply:
All good, thanks for making sure I did this right!